core/src/main/java/io/grpc/Metadata.java

```diff
@@ -354,12 +356,19 @@ public Trailers() {
 
   /**
    * Marshaller for metadata values that are serialized into ASCII strings that contain only
-   * printable characters and space.
+   * following characters:
+   * <ul>
+   *   <li>Horizontal tab: {@code %x09}</li>
```

Comment:
Sorry, I was a bit more explicit in #266 which was linked to from #725. Notably, we aren't including horizontal tab (because _who uses that?!_) and comma (because we don't know how it will factor in yet).